docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -153,6 +157,69 @@ The histogram shows historical changes in a particular host's risk score(s). To
 [role="screenshot"]
 image::images/data-tables.png[]
 
+[[troubleshoot-host-risk-score]]
+[discrete]
+=== Troubleshooting
+
+During the installation or upgrade process, you may receive the following error messages:
+
+* `Saved object already exists`
+* `Transform already exists`
+* `Ingest pipeline already exists`
+
+In this case, we recommend that you manually delete the host risk score module, and then re-enable it. To manually delete the module:
+
+. Delete the host risk score saved objects:
+.. From the {kib} main menu, go to **Stack Management** -> **Kibana** -> **Saved Objects**.
+.. Delete the saved objects that have the `Host Risk Score - <space-id>` tag.
++
+[role="screenshot"]
+image::images/delete-hrs-saved-objects.png[Delete host risk score saved objects]
+.. Delete the `Host Risk Score - <space-id>` tag.
++
+[role="screenshot"]
+image::images/delete-hrs-tag.png[Delete host risk score tag]
+. Stop and delete the host risk score transforms. You can do this through the {kib} UI or using the {ref}/stop-transform.html[Stop transform API] and {ref}/delete-transform.html[Delete transform API].
+** To delete the host risk score transforms through the {kib} UI:
```

Comment:
```suggestion
** To delete the host risk score transforms through the {kib} UI:
```
My understanding is that this technically should be "via" or "using", not "through" (though through is often misused in this way).